.github/workflows/docs.yml

```diff
@@ -42,9 +42,8 @@ jobs:
 
     - name: Build the documentation
       run: |
-        mkdir build
-        cd build
-        python3 ../docs/generate_docs.py
+        cmake -B build -DUMF_TESTS_FAIL_ON_SKIP=ON
```

Comment:
Why this flag?

Reply:
it's outdated, I think (or displayed wrong in GH)